doc/OPTIONS

```diff
@@ -180,6 +180,7 @@ using --show=left you normally need to supply the --format option too.
 --show=types				or in old custom format
 
 Shows the formats/types of input hashes contained in the input file.
+See doc/SHOW_FORMATS_OPTION.md
```

Comment:
File renamed!

Reply:
I fixed this and rewrote the commit to include the change.